src/main/resources/config/application-dev.yml

```diff
@@ -106,6 +106,10 @@ jhipster:
 
 # Properties to be exposed on the /info management endpoint
 info:
+    tutorial-course-groups: # Names of the tutorial groups that will be automatically added to newly created users
+        students: 'artemis-artemistutorial-students'
+        tutors: 'artemis-artemistutorial-tutors'
```

Comment:
do we really need tutors here? The tutorial is mainly meant for students.
There should only be a few instructors and tutors and it might be fine, if those are added manually

Reply:
I agree on that, as I don't think tutor's and instructor's group assignment is essential. But as these properties are optional and can simply be removed from the property file, I would leave that possibility too.

In this case, the "tutors" and "instructors" properties could be excluded from the `application-dev.yml` file and included only if needed.

If there is a "properties-policy" I don't know, I can of course leave these two out and refactor the methods.